pkg/opensearch/client/search_request.go

```diff
@@ -618,7 +618,7 @@ func (b *aggBuilderImpl) Stats() AggBuilder {
 }
 
 // TraceList sets the "aggs" object of the query to OpenSearch for the trace list
-func (b *aggBuilderImpl) TraceList() AggBuilder {
+func (b *aggBuilderImpl) TraceList(spanLimit int) AggBuilder {
```

Comment:
Then this is technically not just the "span" limit, right? Maybe we could make it more clear in the form too, that we're also setting the trace limit and not just span limit, by renaming the input to `Size`, or something similar? It's a common parameter in OpenSearch queries so I think the users would understand what it means too

Reply:
renamed!